### ISSUE TITLE
Problems with CPP14.g4

### ISSUE THREAD

**Issue by dmedme** [antlr/grammars-v4#1577]
1. The Lexer does not recognise multi-line strings. I have a fix for this if anyone is interested.

2. A C++ source file with just the following contents
/*
 * A comment
 */
static const char * whatever = "A multi-line string\n\
A bit more of it\n";

gives "no viable alternative at input 'staticconstchar'"

Grun displays a tree with ALL of the input (including that beyond where it has given up) classified as I would expect, so it isn't clear to me why it has choked.

**Comment by KvanTTT** [comment]
> 1. The Lexer does not recognize multi-line strings. I have a fix for this if anyone is interested.

Yes, we are interested in it very much :) You are welcome to make a pull request.

> gives "no viable alternative at input 'staticconstchar'"

Maybe support of multiline strings with backslashes are not very good and correct?

**Comment by dmedme** [comment]
Yes, it was the backslash handling. I will endeavour to submit a 'pull' request.

**Comment by dmedme** [comment]
Fallen at the first hurdle. I am not permitted to create the branch Issue-#1577-CPP14.g4 as a precursor to creating the pull request.

**Comment by KvanTTT** [comment]
You can do whatever you want, but at first, you have to create a [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) of this repo.
